src/openmrs-resource/encounter.resource.ts

```diff
@@ -2,17 +2,26 @@ import { openmrsObservableFetch } from "@openmrs/esm-api";
 import { Observable } from "rxjs";
 import { map } from "rxjs/operators";
 import { Form, toFormObject } from "./form.resource";
+import { toOmrsDateString } from "../utils/omrs-dates";
 
 export function getPatientEncounters(
-  patientUuid: string
+  patientUuid: string,
+  v = "full",
```

Comment:
Do we need `full` representation in this case? I have a feeling it will increase the request time and size.